pkg/controller/atlasdatabaseuser/atlasdatabaseuser_controller.go

```diff
@@ -281,29 +260,3 @@ func (r *AtlasDatabaseUserReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Watches(&corev1.Secret{}, watch.NewSecretHandler(&r.DeprecatedResourceWatcher)).
 		Complete(r)
 }
-
-func managedByAtlas(ctx context.Context, atlasClient *mongodbatlas.Client, projectID string, log *zap.SugaredLogger) customresource.AtlasChecker {
```

Comment:
This will collide with the db user translation layer PoC, loser PR will have a fun rebase time

Reply:
that's fine :)

Reply:
@josvazg if you want, i can make the experiment of rebasing this on top of the translation PR for the kicks. I am used to rebasing from my past.

Reply:
it is fine, let's just leave it with the flow and merge first whichever makes more sense